server/block/redstone_ore.go

```diff
@@ -0,0 +1,39 @@
+package block
+
+import "github.com/df-mc/dragonfly/server/item"
+
+// RedstoneOre is a common ore.
+type RedstoneOre struct {
```

Comment:
I believe the server needs to handle the state of redstone becoming active when interacted with?

Reply:
Missing lit state